coldboot.js

```diff
@@ -48,6 +49,20 @@ window.coldboot = (function(){
         return {isAwait:false};
     }
 
+    function isRouteTarget(target){
+        var attr = target.getAttributeNames();
+        if(attr.indexOf(route) >= 0){
+            return {isRoute:true,target:target};
+        }
+        var elements =  document.querySelectorAll(`[${route}]`)
+        for(var i=0;i < elements.length;i++){
+            if(elements[i].contains(target)){
+                return {isRoute:true,target:elements[i]};
```

Comment:
should probably add a null check here to avoid sending users to /undefined or reloading the page.